seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/main/java/io/seata/spring/boot/autoconfigure/SeataTCCFenceAutoConfiguration.java

```diff
@@ -36,9 +33,8 @@
  * @author kaka2code
  */
 @ConditionalOnExpression("${seata.enabled:true}")
-@ConditionalOnBean(type = {"javax.sql.DataSource", "org.springframework.transaction.PlatformTransactionManager"})
```

Comment:
If the bean doesn't exist here the following tccFenceConfig should report an error, I think it's important to keep this line of code

Reply:
> If the bean doesn't exist here the following tccFenceConfig should report an error, I think it's important to keep this line of code

Yes, you are right. I reverted the code.